com.sap.cloud.lm.sl.cf.database/src/test/java/com/sap/cloud/lm/sl/cf/database/migration/executor/type/DatabaseTypeSetterFactoryTest.java

```diff
@@ -0,0 +1,114 @@
+package com.sap.cloud.lm.sl.cf.database.migration.executor.type;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+class DatabaseTypeSetterFactoryTest {
+    private static final String EXISTING_DATABASE_TYPE_SETTER = "bool";
+
+    private static final String NON_EXISTING_DATABASE_TYPE_SETTER = "test";
+
+    private DatabaseTypeSetterFactory databaseTypeSetterFactory;
```

Comment:
You don't need to have this field. Fields are used when you want to share state between methods, but you're changing the value in each method, so turn this into a local variable.